### ISSUE TITLE
Error in sort.int just after "All findings:" result report.

### ISSUE THREAD

**Issue by BorisVSchmid** [caitiecollins/treeWAS#12]
Hi Caitie,

getting the following error when running with the attached dataset. 

> 	#################### 
> 	## All findings:  ## 
> 	#################### 
> Number of significant loci: [1] 0
> Error in sort.int(x, na.last = na.last, decreasing = decreasing, ...) : 
>   'x' must be atomic

To make replication of the error easy, here is the data:

[mut-nonmut-coregenome.csv.txt](https://github.com/caitiecollins/treeWAS/files/1433520/mut-nonmut-coregenome.csv.txt)
[logpheno-vector.csv.txt](https://github.com/caitiecollins/treeWAS/files/1433522/logpheno-vector.csv.txt)

and here is the R-code

```
rawpheno <- read.csv(file="logpheno-vector.csv.txt", header=TRUE, sep=",")
pheno <- rawpheno[,2]
names(pheno) <- rawpheno[,1]

rawgeno <- read.csv(file="mut-nonmut-coregenome.csv.txt", header=TRUE, sep=",")
coregeno <- as.matrix(rawgeno)

treeWAS(coregeno,pheno)

```

**Comment by caitiecollins** [comment]
Hi Boris, 

Good to hear from you. Sorry for the delay-- been a bit busy travelling about. 

This one was a little bit sneaky. I ran the lines you sent me and didn't get the error you found. Unfortunately I didn't find any significant results, but that was it. 

That said, is there any chance that you tried to use either the print.treeWAS() or write.treeWAS() functions? It turns out there was a bug in those that would give the error message you sent me when they encountered zero significant findings. I've fixed that now. 

Could that have been the issue? 

If you reinstall and re-run now, I think that should work now.

Best wishes, 
Caitlin.
